cmd/faFormat/testdata/expected.NoGap.bed

```diff
@@ -3,4 +3,4 @@ TrimName Test	0	4
 ToUpperTest	0	4
 NoGapTest	0	3
 NoGapBedTest	0	3
```

Comment:
Only "NoGapBedTest 6 10" has the unique Name field, but not "NoGapBedTest 0 3", "NoGapTest 0 3" etc. Is this expected behavior?